pkg/http/testserver/init.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"os"
+	"strconv"
+)
+
+func init() {
+	if os.Getenv("FIX_LISTEN_PID") == "1" {
+		os.Setenv("LISTEN_PID", strconv.Itoa(os.Getpid()))
+		os.Unsetenv("FIX_LISTEN_PID")
+	}
```

Comment:
What's this used for? Debugging?

Reply:
Maybe I should have the comment from https://github.com/coreos/go-systemd/blob/7d375ecc2b092916968b5601f74cca28a8de45dd/examples/activation/listen.go#L29 there :)